packages/core/src/App/Containers/NotificationsDialog/__tests__/notifications-list.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { render, screen } from '@testing-library/react';
+import NotificationsList from '../notifications-list';
+
+jest.mock('App/Components/Routes', () => ({ BinaryLink: 'MockedBinaryLink' }));
+
+describe('NotificationsList', () => {
+    const renderComponent = (mock_store_override = {}) => {
+        const mock_store = mockStore({
```

Comment:
do you need to call this mockStore each time?
otherwise you can define it in upper scope.

Reply:
in case we need to override some keys, we will be spreading it from `mock_store_override`. so its required during each run.